twtr/grails-app/assets/javascripts/app/templates/search.tpl.htm

```diff
@@ -27,10 +27,11 @@
             <tr>
                 <th class="column-header">Tweeter Handle</th>
                 <th class="column-header">Tweet Messages</th>
+                <th class="column-header">Date Created</th>
```

Comment:
For consistency, we should use the same column header for Date Created on messages (Tweets).  From a UX standpoint, I used "Date/Time Tweeted" as the header.  What are your thoughts?

Also, the header for "Tweet Messages" shouldn't be plural, rather singular since each row only represents a single tweet.